uniflow/extract/server.py

```diff
@@ -28,9 +28,14 @@ def __init__(self, config: Dict[str, Any]) -> None:
         self._flow_cls = FlowFactory.get(self._config.flow_name, flow_type=EXTRACT)
         self._num_thread = self._config.num_thread
         self._flow_queue = Queue(self._num_thread)
+        args = []
+        if self._config.guided_prompt_template:
+            args.append(self._config.guided_prompt_template)
+        if self._config.model_config:
+            args.append(self._config.model_config)
```

Comment:
This is a bad practice, you should use kwargs. What if people messed the order here?